contrib/src/main/java/com/twelvemonkeys/contrib/tiff/TIFFUtilities.java

```diff
@@ -517,6 +664,7 @@ public void rotate(int degree) {
     /**
      * TODO: Temporary clone, to be removed after TMI204 has been closed
      */
+
```

Comment:
Unnecessary blank line. Aslo, #204 is now closed, so the above TODO should be removed/changed.